### ISSUE TITLE
Add "Work Orders - Communications" page 

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#10779]
> What application are you using?

Data Tracker

> Describe the problem.

Add new page under Work Orders " Communications "
Want all the elements of the Technician page, remove DOSA and Signal Attachments.
Add button for Time Log.
If possible add a button or option for " Pending " and " On Hold "  so it shows as Pending or On Hold where it normally says Assigned.

> Describe the outcome you'd like to see when this feature is implemented.

Communications Team will have the flexibility to add time to a work order that is not assigned to them.
The ability to set a work order to Pending or On Hold if the Communications team is waiting on material, work that needs to be done by another group within AMD or outside of AMD or contractor scheduling

> Describe any workarounds you currently have in place or alternative solutions you've considered.

If I need to add time to a work order that is assigned to a team member, I have to reassign that work order to me, add my time, then reassign the work order back to the team member. Most of the time all team members will need to add time to a work order.

As for Pending or On Hold, we have to add that to the Notes in the time log section.

> Is there anything else we should know?

Need this as soon as possible.
I am having to create work orders for Communications due to all the damaged FOC that is occurring. Need to be able to log time for the Com outages. 

> Requested By
Harrell M.

Request ID: DTS22-105675

**Comment by dianamartin** [comment]
# 1/27
- [x] Created new page `Work Orders Communications`
- [x] Added table from "Non-Technician Work Orders"
- [x] Updated table Source filter to be only "Work group = Communications"
- [x] Added menu that had filtered by status 
- [ ] Review Work Order Details page to include all Work Order Technician elements

Work Order Details section | Technician | Communications | Action 
-- | -- | -- | -- | 
Map | X |  |
Menu nav | X |  |
Work Order Details fields | X | | 
Comments | X | X | 
Attachments | X | X | 
DOSA | X | -- |
Time log | X | X | 
Scheduled Work Tasks | X | X | 
Completed Work | X | X | 
Inventory Items | X | X | 
Preventative Maintenance | X | X |

**Comment by dianamartin** [comment]
# 1/27
Emailed Harrell, Brian C, Christina
>Hi Harrell, 
I wanted to let you know I was able to finally work on your request for a “Work Order | Communications” page. To recap: 

### Features Requested
- Add new page under Work Orders " Communications "
- Want all the elements of the Technician page 
  - [-] remove DOSA
  - [-] remove Signal Attachments
  - [+] Add button for Time Log
### New Feature - "On Hold"
- You asked for:
  - If possible, add a button or option for " Pending " and " On Hold " so it shows as Pending or On Hold where it normally says Assigned.
- This needs to be scoped with the rest of the AMD group to update the “Work Order Status” to include “On Hold” and other fields relating to On Hold, like “On Hold Date”, etc..

I did the following: 
- Created new page [Work Orders Communications](https://atd.knack.com/amd#work-orders-communications/) (<-- click here to preview, there isn’t a current link to it from the Data Tracker yet)
- Added table from "Non-Technician Work Orders"
- Updated table Source filter to be only "Work group = Communications"
- Added menu that had filtered by status
- Review Work Order Details page to include all Work Order Technician elements

>Need your feedback on
- Did you want to have the Preventative Maintenance section that’s on the “Work Order – Technicians” page?
- The “Follow-up button” isn’t on the Non-Tech and Communications page right now. This will require an enhancement to have the capability that the “Work Order – Technician” has
  - Work Around: go to the “Work Order – Technician” page, remove filter for “Technicians” and go into the Work Order details page and use the “Follow-up button” from there.
(Inserted screen shot from above)

**Comment by ChristinaTremel** [comment]
# 2/10 Meeting with Harrell

- Walked through the fields added/removed on the work order details page
- Walked through the new Communications Work Order page
- Follow-up button isn't on the non-tech and communication page right now - is this something that Harrell needs to have? 
- **Will need to submit a new request** - Harrell will like to have the Preventative Maintenance form but it's going to be a different set of drop downs. Examples:
     - Cleaning Fiber
     - Checking Fiber with Power Meters
     - Power Meter with short text field to record the measurement
     - Harrell can look at the existing Preventative Maintenance form and create a list of the items that they would PM in Communications
     
**- Harrell approves the changes and is going to submit tickets for some future needs like the preventative maintenance form and on hold date tracking**

**Comment by ChristinaTremel** [comment]
@dianamartin last task for this issue is to: 

- [ ] Add **Work Orders - Communication** page to work orders drop down

**Comment by dianamartin** [comment]
Added "Work Order | Communications" has been added to the Work Order drop down list

![image](https://user-images.githubusercontent.com/21112975/218191587-11e79a93-8791-412d-a602-0daf0b3b2fc0.png)

**Comment by ChristinaTremel** [comment]
Emailed Harrell that the "Work Order - Communications" was added to the work order drop down.
